### DISCUSSION TITLE
Is there a way to turn off the keyboard controls offered by useSliderThumb && useSlider

### DISCUSSION THREAD

**Discussion by amrmetwalyy** [adobe/react-spectrum#4550] [General]
I have an issue with useSlider && useSliderThumb having their keyboard controls, which interfere with my current keyboard controls. The only way to avoid the issue is to click away from the Thumb whenever it's selected. I wonder if that should be a bug.

**Comment by snowystinger** [comment]
I don't think we want to disable the keyboard controls, those are there for keyboard users who have focused a slider. Can you disable your keyboard controls when focus is on the slider?
Otherwise, can you explain more about your use case? Maybe we'll have some other suggestions.

**Reply by amrmetwalyy** [reply]
How can I manually disable the focus on the slider after I'm done dragging? I want it to where it only focuses if I'm dragging the slider. That way I can avoid the keyboard controls being overridden by the ones used in useSlider && useSliderThumb

**Reply by snowystinger** [reply]
I really need to know more about the use case before I can recommend anything. It's sounding like the slider isn't actually the real slider, merely the visual representation of one and the real slider is somewhere else, maybe higher up in the dom tree.

**Reply by amrmetwalyy** [reply]
The slider in the video player retains its focus and remains highlighted in blue after dragging it with the mouse. That activates the keyboard shortcuts implemented by the useSlider && useSliderThumb. However, this causes a conflict with my custom keyboard controls implemented using react-hotkeys-hook. The keyboard controls, such as the up/down arrow keys and right/left arrows, don't work while the slider is in focus. To regain control of the keyboard, I have to click away from the video player (video slider).

**Reply by snowystinger** [reply]
I think after dragging it with the mouse, it would be appropriate for you to move focus to some other more appropriate element `element.focus()` Then the keyboard events wouldn't go to the slider. However, if the user purposefully keyboard navigates to the slider, then I think it's expected that the slider responds to the keyboard events. If the user no longer wants to interact with the slider, then they would move focus off to the element they do want to interact with.

**Reply by amrmetwalyy** [reply]
I tried your solution and need help getting it to work. I have a problem where the focus is persistently retained by the thumb regardless if I have my logic in onChangeEnd prop func. There has to be a solution; any ideas are much appreciated!

**Reply by snowystinger** [reply]
looks like there is a bug, you are correct something is yanking it back https://codesandbox.io/s/spring-glitter-329tzn?file=/src/App.js

I've submitted an issue https://github.com/adobe/react-spectrum/issues/4565

In the meantime, you could give a 50ms or so timeout before moving the focus to the button. I suspect there is a raf that is pulling it back, so that should be plenty of time.

**Reply by amrmetwalyy** [reply]
Thank you so much!
